web3.js

```diff
@@ -36,6 +36,19 @@ function load(opts) {
   return new Web3(provider(opts.provider))
 }
 
+/**
+ * Returns the provider(s) based on .ararc network id.
+ * @param  {Object} web3
+ * @return {String|Array}
+ */
+function getProvider(web3) {
+  const { network_id } = web3
```

Comment:
Does ini not allow camelCase? Totally cool with this if so. :)

Reply:
(Alternatively, you could do `{ network_id: networkId } = web3` to maximize consistency. ;))

Reply:
camel case doesn't belong on JSON or INI so 👍 for the rename when destructuring

Reply:
There is no real standard but from the `.ini` files I can see they are either snake case or no case, ie `drivername`. Converted to camelcase ;-)